sim/deploy/run.py

```diff
@@ -21,7 +21,7 @@
 
 from sim.deploy.config import RobotConfig
 from sim.env import stompy_mjcf_path
-from sim.stompy.joints import StompyFixed
+from sim.stompy2.joints import StompyFixed
```

Comment:
Let's remove old stompy and keep stompy2 only as stompy

Reply:
done

Reply:
not done